packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -318,96 +370,54 @@ See [request properties](#objects-in-the-request-from-okta) for full details.
 }
 ```
 
-## Sample response
+## Sample progressive enrollment responses
 
-The external service responds to Okta indicating whether to accept the end user's self-registration or profile update. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to self-register or to update their profile with Okta.
+The external service responds to Okta indicating whether to accept the end user's profile update. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to update their profile with Okta.
 
 See [response properties](#response-objects-that-you-send) for full details.
 
-```javascript
-// Registration Inline Hook code to parse the incoming Okta request
-
-app.post('/registrationHook', async (request, response) => {
-  console.log();
-
-  var returnValue = {};
-
-  if (request.body.requestType === 'progressive.profile') {
-    // For example, 'employeeNumber' is an additional attribute collected after end user registration.
-    console.log('Employee number added to profile ' + request.body.data.context.user.profile['login'] + ': ' + request.body.data.userProfileUpdate['employeeNumber']);
-    var employeeNumber = request.body.data.userProfileUpdate['employeeNumber'];
-    if (employeeNumber && employeeNumber.length === 4) {
-      returnValue = {
-        'commands':[
-          {
-            type: 'com.okta.user.progressive.profile.update',
-            value: {
-              'employeeNumber': employeeNumber,
+### Sample progressive enrollment response using data.UserProfileUpdate
+
+The following sample response shows an end user successfully adding their employee number to their profile:
```

Comment:
Somewhat pedantic, but it's the code that's adding the employeeNumber to the profile. Maybe something like "The following sample response shows the addition of an employee number to an end user's profile."

Reply:
Thank you, Brian :)